docs/develop-guide.rst

```diff
@@ -336,38 +336,101 @@ Start the CLI to connect to the server and run SQL queries:
+   cp  ./presto/presto-native-execution/_build/release/presto_cpp/main/presto_server ./${package_name}/bin
+   cp  ./presto/presto-native-execution/presto_cpp/main/lib/libcider.so  ./${package_name}/lib
+   cp	/usr/local/lib/libantlr4-runtime.so.4.9.3	./${package_name}/lib
+   cp	/lib/x86_64-linux-gnu/libre2.so.5	./${package_name}/lib
+   cp	/lib/libprotobuf.so.32	./${package_name}/lib
+   cp	/usr/local/lib/libLLVM-9.so	./${package_name}/lib
+   cp	/usr/local/lib/libtbb.so.12	./${package_name}/lib
+   cp	/lib/x86_64-linux-gnu/libdouble-conversion.so.3	./${package_name}/lib
+   cp	/lib/x86_64-linux-gnu/libglog.so.0	./${package_name}/lib
+   cp	/lib/x86_64-linux-gnu/libgflags.so.2.2	./${package_name}/lib
+   cp	/lib/x86_64-linux-gnu/libsodium.so.23	./${package_name}/lib
+   cp	/lib/x86_64-linux-gnu/libssl.so.1.1	./${package_name}/lib
+   cp	/lib/x86_64-linux-gnu/libcrypto.so.1.1	./${package_name}/lib
+   cp	/lib/x86_64-linux-gnu/libboost_context.so.1.71.0	./${package_name}/lib
+   cp	/lib/x86_64-linux-gnu/libevent-2.1.so.7	./${package_name}/lib
+   cp	/lib/x86_64-linux-gnu/libsnappy.so.1	./${package_name}/lib
+   cp	/lib/x86_64-linux-gnu/libboost_regex.so.1.71.0	./${package_name}/lib
+   cp	/lib/x86_64-linux-gnu/libboost_locale.so.1.71.0	./${package_name}/lib
+   cp	/lib/x86_64-linux-gnu/libboost_log.so.1.71.0	./${package_name}/lib
+   cp	/lib/x86_64-linux-gnu/libboost_filesystem.so.1.71.0	./${package_name}/lib
+   cp	/lib/x86_64-linux-gnu/libboost_program_options.so.1.71.0	./${package_name}/lib
+   cp	/lib/x86_64-linux-gnu/libboost_thread.so.1.71.0	./${package_name}/lib
+   cp	/lib/x86_64-linux-gnu/libedit.so.2	./${package_name}/lib
+   cp	/lib/x86_64-linux-gnu/libunwind.so.8	./${package_name}/lib
+   cp	/lib/x86_64-linux-gnu/libicui18n.so.66	./${package_name}/lib
+   cp	/lib/x86_64-linux-gnu/libicuuc.so.66	./${package_name}/lib
+   cp	/lib/x86_64-linux-gnu/libbsd.so.0	./${package_name}/lib
+   cp	/lib/x86_64-linux-gnu/libicudata.so.66	./${package_name}/lib
+   tar -czvf ${package_name}.tar.gz ${package_name}
+
+7.3 Unzip the Prestodb package and enter the unzip package
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 ::
 
-   cp /path/to/cider/build/function/ExtensionFunctions.ast   /path/to/cider/function
-   cp /path/to/cider/build/function/RuntimeFunctions.bc   /path/to/cider/function
+   tar -zxvf Prestodb.tar.gz
+   cd Prestodb
 
-(7) Set the LD_LIBRARY_PATH environment variable include the lib folder.
+7.4 Set the LD_LIBRARY_PATH environment variable include the lib folder
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 ::
 
-   export LD_LIBRARY_PATH=/path/to/cider/lib:$LD_LIBRARY_PATH
+   export LD_LIBRARY_PATH=./lib:$LD_LIBRARY_PATH
 
-(8) You may also need include the libjvm.so in your LD_LIBRARY_PATH if
-    it is not
+7.5 run presto_server
+~~~~~~~~~~~~~~~~~~~~~
 
 ::
-
-   export LD_LIBRARY_PATH= $JAVA_HOME/jre/lib/amd64/server/:$LD_LIBRARY_PATH
+   
+   ./bin/presto_server
```

Comment:
will work? Need run with parameter point to etc folder?

Reply:
fix